setup.py

```diff
@@ -0,0 +1,31 @@
+import setuptools
+
+setuptools.setup(
+	name='apple-health-exporter',
+	version='1.0.0',
+	author='Milind Ganjoo',
+	author_email='milind.ganjoo@gmail.com',
+	description='Python script to export Apple Health dump file to a data frame for analysis.',
+	long_description="""
+# Apple Health Data Exporter
+
+This Python 3 script takes a dump of Apple Health data (the `export.zip` file
+generated through an Apple Health data export) and saves a subset of fields and
+records to a [feather](https://github.com/wesm/feather) file. This can then be
+read by both Python and R.
+	""",
+	long_description_content_type="text/markdown",
+	url='https://github.com/mganjoo/apple-health-exporter',
+	project_urls = {
+		"Bug Tracker": "https://github.com/mganjoo/apple-health-exporter/issues"
+	},
+	license='MIT',
+	packages=['apple_health_exporter'],
+	package_dir={'.':''},
+	scripts=['apple_health_exporter/export.py'],
```

Comment:
Instead of `scripts`, you can set up [`entry_points`](https://setuptools.pypa.io/en/latest/userguide/entry_point.html#console-scripts):

```suggestion
	entry_points={
		'console_scripts': [
			'apple-health-exporter=apple_health_exporter.export:export',
		]
	},
```

As mentioned in the main comment, this will allow pip to simply install a binary executable that calls the `export()` function in the `export` module, meaning you could run the pip binary simply by calling `apple-health-exporter` on the command line.